### PR TITLE
[DOCKER][MME] configuring MME to run stateful in OAI deployment

### PR DESCRIPTION
**Pull request by rdefosse**
Signed-off-by: Raphael Defosseux <raphael.defosseux@openairinterface.org>

## Summary

Based on Shruti's feedback on the last weekly CONV-MME call.

## Test Plan

CI on OAI side should pass.

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by rdefosse** [comment]
MAGMA-OAI-TESTS

**Review by ulaskozat — APPROVED** [review]
LGTM

**Review by ssanadhya — APPROVED** [review]

**Review by 119Vik — APPROVED** [review]

**Review by Vyas20 — APPROVED** [review]

**Review by Vyas20 — APPROVED** [review]

**Review by Vyas20 — APPROVED** [review]

File: lte/gateway/docker/mme/Dockerfile.ubuntu18.04 [hunk 1]
```diff
@@ -275,7 +275,9 @@ RUN cp $MAGMA_ROOT/lte/gateway/docker/mme/configs/mme.conf .
 RUN cp $MAGMA_ROOT/lte/gateway/docker/mme/configs/mme_fd.conf .
 
 # Create running dirs
-WORKDIR /var/opt/magma
+WORKDIR /var/opt/magma/configs
```
Comment (mattymo) [inline lte/gateway/docker/mme/Dockerfile.ubuntu18.04:278]: Here you use /var/opt/magma/configs and on line /var/opt/magma/config
Which is right?

Comment (rdefosse) [reply lte/gateway/docker/mme/Dockerfile.ubuntu18.04:278]: Hi @mattymo 

I just followed what @ssanadhya told me  in a [slack message](https://magmacore.slack.com/archives/C01B32Q9L76/p1612977897017200)

Comment (ssanadhya) [reply lte/gateway/docker/mme/Dockerfile.ubuntu18.04:278]: Good catch @mattymo ! the override configs should always be in `/var/opt/magma/configs`

File: lte/gateway/docker/mme/Dockerfile.ubuntu18.04 [hunk 2]
```diff
@@ -374,7 +376,9 @@ COPY --from=magma-mme-builder $MAGMA_ROOT/lte/gateway/docker/mme/configs/mme.con
 COPY --from=magma-mme-builder $MAGMA_ROOT/lte/gateway/docker/mme/configs/mme_fd.conf .
 
 # Create running dirs
-WORKDIR /var/opt/magma
+WORKDIR /var/opt/magma/config
```
Comment (ssanadhya) [inline lte/gateway/docker/mme/Dockerfile.ubuntu18.04]:
```suggestion
WORKDIR /var/opt/magma/configs
```

**Comment by rdefosse** [comment]
MAGMA-OAI-TESTS

File: lte/gateway/docker/mme/Dockerfile.ubuntu18.04 (hunk 1 above)
Comment (rdefosse) [reply lte/gateway/docker/mme/Dockerfile.ubuntu18.04:278]: Fixed the typo! Thx @mattymo and @ssanadhya

**Comment by ssanadhya** [comment]
@rdefosse, can you rebase this PR on the latest master to fix the `feg-build` CI check?

**Comment by rdefosse** [comment]
MAGMA-OAI-TESTS

**Comment by rdefosse** [comment]
@ssanadhya done

**Pull request merged**